corehq/warehouse/transforms/sql/domain_dim.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO {{ domain_dim }} (
```

Comment:
I realize this shouldn't be `INSERT`, but I'm talking with javier and lu about the possibility of upgrading PG in order to take advantage of `UPSERT`. I think that `UPSERT` will make the code simpler and easier to understand, so if the LOE isn't too high, i'd vote to leave these as insert until we upgrade